pkg/capabilities/embedded/eopa/v0.100.5.json

```diff
@@ -0,0 +1 @@
+404: Not Found
```

Comment:
You'll want to address this, I'm sure :) But it's actually a good reminder that we need to test somewhere that the files downloaded are legit.

Reply:
2 steps ahead of you 😉 646a37db8db27a6f6661f280c37962cd8a806a99